src/v/archival/archival_policy.cc

```diff
@@ -285,7 +285,9 @@ static ss::future<std::optional<std::error_code>> get_file_range(
     upl->final_file_offset = fsize;
     upl->base_timestamp = segment->index().base_timestamp();
     upl->max_timestamp = segment->index().max_timestamp();
-    if (!end_inclusive && segment->offsets().get_base_offset() == begin_inclusive) {
+    if (
+      !end_inclusive
+      && segment->offsets().get_base_offset() == begin_inclusive) {
```

Comment:
before merging we'll want to fixup the clang-format changes into a single commit. but plus one to splitting them out from the commit with the main changes to keep it simpler to read.

>clang-format (forgot .cc)

also, in case you were doing this manually, you can also invoke `tools/format-cc` to do this tree-wide.